engine/src/main/scala/cromwell/engine/workflow/lifecycle/finalization/CopyWorkflowLogsActor.scala

```diff
@@ -42,7 +42,7 @@ class CopyWorkflowLogsActor(override val serviceRegistryActor: ActorRef, overrid
   }
 
   def deleteLog(src: Path) = if (WorkflowLogger.isTemporary) {
-    sendIoCommand(GcsBatchCommandBuilder.deleteCommand(src))
+    sendIoCommand(DefaultIoCommandBuilder.deleteCommand(src))
```

Comment:
This was actually doing the right thing of deleting the local log file, though for unintuitive reasons. 🙂